manager/catalog_manager.go

```diff
@@ -26,10 +36,11 @@ func (i *arrayFlags) Set(value string) error {
 }
 
 var (
-	refreshInterval = flag.Int64("refreshInterval", 60, "Time interval (in Seconds) to periodically pull the catalog from git repo")
-	logFile         = flag.String("logFile", "", "Log file")
-	debug           = flag.Bool("debug", false, "Debug")
-	validate        = flag.Bool("validate", false, "Validate catalog yaml and exit")
+	refreshInterval  = flag.Int64("refreshInterval", 60, "Time interval (in Seconds) to periodically pull the catalog from git repo")
+	logFile          = flag.String("logFile", "", "Log file")
+	debug            = flag.Bool("debug", false, "Debug")
+	validate         = flag.Bool("validate", false, "Validate catalog yaml and exit")
+	catalogURLBranch = flag.String("catalogURLBranch", "", "Branch name")
```

Comment:
Was this accidentally left in?  `catalogUrl` can have several different URLs in it so specifying a single branch doesn't make sense, and we only need to support branches with the new config file option.  Also it's never used and overwritten by a loop farther down.